src/clover/load/load.py

```diff
@@ -798,14 +809,35 @@ def process_device_hourly_usage(
 
         logger.info("Hourly usage profile for %s successfully calculated.", device.name)
 
+        #Retrieve device laod time
+        n = device.load_time
```

Comment:
## :memo: Variable names
Would be good to have nicer variable names so it's easier to read through and understand what's going on